fs/copy.py

```diff
@@ -29,11 +29,43 @@ def copy_fs(src_fs, dst_fs, walker=None):
         in ``src_fs``. Set this if you only want to consider a sub-set
         of the resources in ``src_fs``.
     :type walker: :class:`~fs.walk.Walker`
+    :returns: a list with the copied files dst_path.
 
     """
-    copy_dir(src_fs, '/', dst_fs, '/', walker=walker)
+    return copy_dir(src_fs, '/', dst_fs, '/', walker=walker)
 
 
+def copy_fs_if_newer(src_fs, dst_fs, walker=None):
+    return copy_dir_if_newer(src_fs, '/', dst_fs, '/', walker=walker)
+
+
+def _source_is_newer(src_fs, src_path, dst_fs, dst_path):
+    """
+       Determine if source file is newer than destination file.
+
+       :param src_fs: Source filesystem.
+       :type src_fs: :type src_fs: FS URL or instance
+       :param src_path: A path to a directory on ``src_fs``.
+       :type src_path: str
+       :param dst_fs: Destination filesystem.
+       :type dst_fs: FS URL or instance
+       :param walker: A walker object that will be used to scan for files
+           in ``src_fs``. Set this if you only want to consider a sub-set
+           of the resources in ``src_fs``.
+       :type walker: :class:`~fs.walk.Walker`
+
+       """
+    try:
+        if not dst_fs.exists(dst_path):
+            return True
+        else:
+            nmsp = ('details', 'modified')
+            return src_fs.getinfo(src_path, nmsp).modified > dst_fs.getinfo(dst_path, nmsp).modified
+    except Exception as ex:
```

Comment:
Yes, in case of problems (eg cannot find the modified time), i just execute the copy (i.e. return True).
Im my last commit on this PR branch i catch the FSError exception insted of Exception